scripts/ethbuild.sh

```diff
@@ -223,7 +238,12 @@ do
 			exit 1
 		fi
 
-		make -j${MAKE_CORES}
+		if [[ $USEXCODE -eq 0 ]]; then
+			make -j${MAKE_CORES}
+		else
+			xcodebuild
```

Comment:
No, this change won't affect the MacOSX build slave because it uses "xcodebuild" only when you specify the new "-xcode" option. Otherwise it still uses "make".